libs/react-helpers/src/hooks/use-logger.ts

```diff
@@ -0,0 +1,27 @@
+import { useRef } from 'react';
+import { BrowserTracing } from '@sentry/tracing';
+import * as Sentry from '@sentry/browser';
+import type { LocalLogger } from '../lib/local-logger';
+import { localLoggerFactory } from '../lib/local-logger';
+
+interface Props {
+  dsn?: string;
+  application?: string;
+  tags?: string[];
+}
+
+export const useLogger = ({ dsn, ...props }: Props) => {
```

Comment:
question (non blocking): Whats the advantage of providing a hook?  Personally, I'd prefer to keep the component code cleaner and just instantiate the logger directly in the module file.

Reply:
hook gives additional possibility of initialization sentry. It can be useful in other apps than those build on nextjs, where initialization is config file